docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -4,12 +4,113 @@
+
+[[what-is-my-agent-config]]
+=== I need to know the configuration used in the running Agent.
+
+You can inspect the files in the folder from where you ran the Agent.  The elastic-agent.yml file will have an uncommented citation if it is in 'Fleet' mode:
+Management: mode: "fleet"
+
+The action_store.yml has the whole configuration, unencrypted.  Look for 'outputs:hosts' section with elasticsearch location.  This file will also have the current versions of the packages in use by the current configuration.  The version of the Agent is in the download folder / zip.
+
+[[where-is-the-data-agent-is-sending]]
+=== I'm not sure {ingest-manager} app is showing the data my Agent is sending.
+
+I think I have Agent set up and running and there are no errors on the Agent logs and I still don't see data in Kibana.  The recommended way to assess this is to go straight to Elasticsearch data queries, or maybe just use Kibana Discover app.  Look for any data in the index your Agent is sending to (metrics-* for example) and see if the name.host of your host machine shows.  If not, then perhaps the data is getting trapped / blocked in the network or isn't sent off of the Agent host correctly in the first place due to some Firewall or security reason.  While it should be redundant, it is all the same potentially helpful to install the stand-alone Metricbeat (see the Elastic docs for information) to see if that works to send data into ES, which would indicate a bug or problem in the Agent side.
+
+[[i-deleted-my-agent]]
+=== I deleted my agent in the Fleet {ingest-manager} app so now what?
+
+Its ok, be cool: the data is still in Elasticsearch.  But to get the Agent back in the UI, you will need to stop, then re-enroll the Agent on the host and run it again.
+
+[[i-need-to-stop-agent]]
+=== I need to stop the Agent and all processes running on my host
+
+To stop the Agent and relating executables running on my hosts is as easy as stopping the Agent process.  If you installed it as a service on Windows you will need to manage that, as well.  After that, if needed, on Windows you can use Task Manager and shut down ‘Elastic Agent’ which will kill it and the sub-processes it created (beats).  On Linux / macOS you can kill the elastic-agent process running, look with a ps | grep elastic-agent call to get the id to kill
+
+[[i-rebooted-my-host]]
+=== I needed to reboot my host, I'm not sure if the Agent is still running.
+
+If you installed Windows Agent as a service with powershell, the Agent should still be running after a reboot of the host.  On macOS and Linux you will need to start the Agent again from the command line each time.  The ability to enhance support for those operating systems is known, as a temporary fix you could add that start command to a start up profile.
+
+[[what-is-the-endpoint-package]]
+=== The {ingest-manager} app shows an Integration called 'Endpoint' and I want to use it.
```

Comment:
I'd state this more like a problem. What problem does the user encounter when they try to use the endpoint?

Reply:
I don't know for sure and finding out now isn't in scope of our milestone.  I'm open to having this re-written but it was also intentionally left as a playful note and teaser about the coming Integration.  Knowing that I don't know how it will impact things, do you have any specific suggestions we'd want to do?